lib/grunt/index.js

```diff
@@ -26,12 +26,35 @@
 
 'use strict';
 
-module.exports = (grunt, extraArguments) => {
+module.exports = (grunt, pluginOptions) => {
   const chalk = require('chalk');
   const VinylStream = require('./vinyl-stream');
   const Transform = require('stream').Transform;
   const gulpCompilerOptions = {};
   const gulp = require('gulp');
+  const {getFirstSupportedPlatform} = require('../utils');
+
+  let extraArguments;
+  let platforms;
+  if (pluginOptions) {
+    if (Array.isArray(extraArguments)) {
+      extraArguments = pluginOptions;
+    } else if (pluginOptions === 'javascript') {
```

Comment:
It looks like a property name is missing here.

Reply:
Yeah this was a poor choice on my part. Last month when the JS version was first introduced, that was the only other platform. So this was to continue to support that case. See https://github.com/google/closure-compiler-npm/blob/master/docs/grunt.md#js-version-of-closure-compiler